### DISCUSSION TITLE
Windows version is identified as malware by Bitdefender.

### DISCUSSION THREAD

**Discussion by crowstwo** [TheLastGimbus/GooglePhotosTakeoutHelper#267] [I NEED HELP]
I downloaded it from here, scanned it with Bitdefender. Ran it and while it was running, everything crashed, or so I thought.
Bitdefender closed everything I was running and told me it's disinfecting my PC.

It's labeling gpth-windoza.exe as malware but it doesn't say what type of malware.
At the moment I turned off ethernet/wifi on my PC while it's scanning it.
Haven't have something like this happening in 15+ years.

Is the file infected somehow or why did the anti-virus go crazy? It started when the gpth was already running. Not when I started it.

EDIT
Scanning the exe file from my phone with virustotal it gets 3 virus hits. So thread carefully. 
![Screenshot_20231216_112746_Vivaldi Browser](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/35896229/7b7ce68f-fcf2-46bc-a817-60b7cb7f2082)

**Comment by TheLastGimbus** [comment]
Marvelous <3

Why is this? I have no idea. gpth is an .exe, that people download from internet, and then it analyzes and moves a lot of files. Anti-viruses mayyyyyyyyyyy see this as if it's encrypting/stealing them or something??

I have no idea, but gpth, indeed, does not do anything malicious - you can see the whole code, and even the whole pipeline of how it gets built in github actions, and that all exe's in gh releases are uploaded by github bot - i think i really have no way to upload a virus there if i really wanted to :shrug: 

Only one who can spy viruses in here is either Google in Dart lang itself, or Microsoft through github. Which is probable. [insert some fbi meme here]

**Reply by TheLastGimbus** [reply]
related to #114
